app/javascript/controllers/dropdown_controller.js

```diff
@@ -0,0 +1,16 @@
+import Dropdown from '@stimulus-components/dropdown'
+
+export default class extends Dropdown {
+  connect() {
+    super.connect()
+    console.log('Do what you want here.')
+  }
+
+  toggle(event) {
+    super.toggle()
+  }
+
+  hide(event) {
+    super.hide(event)
+  }
```

Comment:
I think you don't need this code. It was in the documentation as an example if you would like to modify the behavior of the Dropdown. Try to remove this file and check if it still works 👍

Reply:
I added a commit to this PR. I hope I did it right and you will see it.